tests/test_meetings.py

```diff
@@ -31,55 +33,72 @@ def test_meetings_landing(self, meetings_page):
         assert meetings_page.nrao_logo.present()
         assert meetings_page.spsp_logo.present()
 
-    def test_filtering(self, meetings_page):
+    def test_filtering(self, meetings_page, driver):
+        search_bar = driver.find_element_by_css_selector('div[data-test-meetings-list-search]')
+        driver.execute_script('arguments[0].scrollIntoView();', search_bar)
+
         default_top_result = meetings_page.top_meeting_link.text
         meetings_page.filter_input.clear()
-        meetings_page.filter_input.send_keys('ea')
+        meetings_page.filter_input.send_keys('f')
+        time.sleep(.5)
```

Comment:
Let's use QUICK_TIMEOUT here

Reply:
Update: QUICK_TIMEOUT is 4 seconds, which is too long.  

Instead, we're finding the skeleton screen loading indicator, and applying here_then_gone